lib/modules/dosomething/dosomething_campaign/includes/Campaign.php

```diff
@@ -22,6 +22,9 @@ class Campaign {
   public $staff_pick;
   public $facts;
   public $solutions;
+  public $pre_step_header;
```

Comment:
Regardless of whether you need to output the "copy" content now or later, at least it starts to set up the structure properly.